trailblazer-context.gemspec

```diff
@@ -23,7 +23,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "minitest"
   spec.add_development_dependency "rake"
-  spec.add_development_dependency "rubocop"
```

Comment:
So how does rubocop works exactly now ?

Reply:
you run it from https://github.com/trailblazer/meta repo.

I will update the readme there  and some rules.

Reply:
Oh cool, I didn't know about this repo 😅